deploy/mobileclient_admin_cluster_role.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: mobileclient-admin-cluster
+rules:
+  - apiGroups:
+      - mobile.k8s.io
+    resources:
+      - mobileclients
+    verbs:
+      - get
+      - list
+      - watch
+      - update
```

Comment:
With the additional context from the comment in jira, I think we can maybe remove the `update` and `patch` here. It seems a little incorrect to give all users these permissions in _all_ namespaces. I think the fact that we have to give even the get/list/watch permissions is maybe a bit of a bug with openshift, in that the console requests them for display, but the default project permissions don't include them.